### PR TITLE
No inventory number

### LINKED ISSUE #201: option to set inventory number to none
because there is no inventory number for less than 250 €

### PR DESCRIPTION
**Pull request by LizzAlice**
fixes #201

### CODE REVIEW

File: devices/migrations/0007_auto_20180327_1440.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.5 on 2018-03-27 12:40
+from __future__ import unicode_literals
```
Comment (octomike) [inline devices/migrations/0007_auto_20180327_1440.py]: can be removed, develop has py3 already merged

File: users/migrations/0012_auto_20180327_1440.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.5 on 2018-03-27 12:40
+from __future__ import unicode_literals
```
Comment (octomike) [inline users/migrations/0012_auto_20180327_1440.py]: s.a.

File: devices/migrations/0007_auto_20180327_1440.py (hunk 1 above)
Comment (octomike) [reply devices/migrations/0007_auto_20180327_1440.py]: ignore that

**Comment by octomike** [comment]
django.po needs fixing, can be merged afterwards

File: users/migrations/0012_auto_20180327_1440.py (hunk 2 above)
Comment (LizzAlice) [reply users/migrations/0012_auto_20180327_1440.py]: ?

Comment (LizzAlice) [reply users/migrations/0012_auto_20180327_1440.py]: Should I fix the django.po, or do you do that?

**Comment by LizzAlice** [comment]
The conflicts were resolved, but there seems to be a problem with the docker tests.

**Comment by LizzAlice** [comment]
Closed because there were to many problems, I'll just start a new branch from develop and copy the changes.

**Pull request closed without merging**